rotkehlchen/globaldb/handler.py

```diff
@@ -1783,15 +1808,68 @@ def resolve_asset(identifier: str, form_with_incomplete_data: bool) -> AssetWith
             )
 
     @staticmethod
-    def get_asset_type(identifier: str) -> AssetType:
+    def resolve_asset_from_packaged_and_store(
+            identifier: str,
+            form_with_incomplete_data: bool,
+    ) -> AssetWithNameAndType:
+        """
+        Reads an asset from the packaged globaldb and adds it to the database if missing or edits
+        the local version of the asset.
+        May raise:
+        - UnknownAsset
+        """
+        asset = GlobalDBHandler().resolve_asset(
+            identifier=identifier,
+            form_with_incomplete_data=form_with_incomplete_data,
+            use_packaged_db=True,
+        )
+        # make sure that the asset is saved on the user's global db. First check if it exists
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            asset_count = cursor.execute(
+                'SELECT COUNT(*) FROM assets WHERE identifier=?',
+                (identifier,),
+            ).fetchone()[0]
+
+        asset_data: Union[EvmToken, Dict[str, Any]]
+        if asset.asset_type == AssetType.EVM_TOKEN:
+            asset_data = cast(EvmToken, asset)
+        else:
+            asset_data = {
+                'identifier': asset.identifier,
+                'asset_type': asset.asset_type,
+                'forked': getattr(asset, 'forked', None),
+            }
```

Comment:
As you have changed it now you are missing information when adding/editing the asset